website/content/en/docs/helm/quickstart.md

```diff
@@ -11,32 +11,42 @@ This guide walks through an example of building a simple nginx-operator powered
 Use the CLI to create a new Helm-based nginx-operator project:
 
 ```sh
-operator-sdk new nginx-operator --api-version=example.com/v1alpha1 --kind=Nginx --type=helm
-cd nginx-operator
+$ mkdir nginx-operator
```

Comment:
Rename this file to `user-guide.md`. I'll submit a PR for a separate quick start document that is much shorter and to the point.

Reply:
would not be better to rename the file in this PR that you will submit? Otherwise, will break the link. WDYT?

Reply:
Yeah, that works too :)